contracts/adapters/FlexibleLeverageStrategyAdapter.sol

```diff
@@ -79,14 +85,17 @@ contract FlexibleLeverageStrategyAdapter is BaseAdapter {
     }
 
     struct ContractSettings {
-        ISetToken setToken;                              // Instance of leverage token
-        ICompoundLeverageModule leverageModule;          // Instance of Compound leverage module
-        IComptroller comptroller;                        // Instance of Compound Comptroller
-        ICompoundPriceOracle priceOracle;                // Compound open oracle feed that returns prices accounting for decimals. e.g. USDC 6 decimals = 10^18 * 10^18 / 10^6
-        ICErc20 targetCollateralCToken;                  // Instance of target collateral cToken asset
-        ICErc20 targetBorrowCToken;                      // Instance of target borrow cToken asset
-        address collateralAsset;                         // Address of underlying collateral
-        address borrowAsset;                             // Address of underlying borrow asset
+        ISetToken setToken;                                     // Instance of leverage token
+        ICompoundLeverageModule leverageModule;                 // Instance of Compound leverage module
+        IComptroller comptroller;                               // Instance of Compound Comptroller
+        IChainlinkAggregatorV3 chainlinkCollateralPriceOracle;  // Chainlink oracle feed that returns prices in 8 decimals for collateral asset
+        IChainlinkAggregatorV3 chainlinkBorrowPriceOracle;      // Chainlink oracle feed that returns prices in 8 decimals for borrow asset
+        ICErc20 targetCollateralCToken;                         // Instance of target collateral cToken asset
+        ICErc20 targetBorrowCToken;                             // Instance of target borrow cToken asset
+        address collateralAsset;                                // Address of underlying collateral
+        address borrowAsset;                                    // Address of underlying borrow asset
+        uint256 collateralDecimalAdjustment;                    // Decimal adjustment for chainlink oracle of the collateral asset. Equal to 28-collateralDecimals
```

Comment:
When we write our deploy scripts, now need to be super careful about these decimal adjustment parameters